app/components/Layout/ExternalLayout.tsx

```diff
@@ -0,0 +1,111 @@
+import { graphql, useFragment } from "react-relay";
+import type { ExternalLayout_session$key } from "ExternalLayout_session.graphql";
+import getConfig from "next/config";
+import Navigation from "components/Layout/Navigation";
+import Footer from "components/Layout/Footer";
+import SiteNoticeBanner from "components/Layout/SiteNoticeBanner";
+import UserProfile from "components/User/UserProfile";
+import GlobalAlert from "./GlobalAlert";
+import { useContext } from "react";
+import { ErrorContext } from "contexts/ErrorContext";
+
+import footerLinks from "data/externalLinks/footerLinks";
+import subHeaderLinks from "data/externalLinks/subHeaderLinks";
+const runtimeConfig = getConfig()?.publicRuntimeConfig ?? {};
+
+interface Props {
+  title?: string;
+  session: ExternalLayout_session$key;
+  leftSideNav?: React.ReactNode;
+}
+
+const ExternalLayout: React.FC<Props> = ({
+  children,
+  title,
+  session: sessionFragment,
+  leftSideNav,
+}) => {
+  const session = useFragment(
+    graphql`
+      fragment ExternalLayout_session on KeycloakJwt {
+        cifUserBySub {
+          ...UserProfile_user
+        }
+      }
+    `,
+    sessionFragment
+  );
```

Comment:
I don't think we're doing any true auth for first slice, so I think it would be fine to leave this out and just set the user to null below (might be easier to unit test)